graphql_api/types/repository/repository.py

```diff
@@ -338,6 +338,11 @@ def resolve_bundle_analysis_enabled(repository: Repository, info) -> bool:
     return repository.bundle_analysis_enabled
 
 
+@repository_bindable.field("coverageEnabled")
+def resolve_coverage_enabled(repository: Repository, info) -> bool:
```

Comment:
Since the return type is optional cause it can be null, can we add Optional to the return type?

Reply:
👍